test/langtools/jdk/javadoc/doclet/testNavigation/TestNavigation.java

```diff
@@ -53,28 +54,67 @@ public TestNavigation() {
     }
 
     @Test
-    public void test(Path ignore) {
-        javadoc("-d", "out",
+    public void testOverview(Path ignore) {
+        javadoc("-d", "out-overview",
```

Comment:
Generally, I would recommend renaming `Path ignore` to `Path base` and then generating a filename for use in the `javadoc` call, such as `base.resolve("api").toString()`. This helps guarantee  the files for each test case are isolated.